features/step_definitions/tournament.rb

```diff
@@ -41,10 +41,14 @@
   visit event_schedule_path(tournament_named tournamentName)
 end
 
-And(/^the texts? (.+) (?:are|is) there\.$/) do |texts_raw|
+And(/^the texts? (.+) (?:are|is)( not)? there\.?$/) do |texts_raw, _not|
```

Comment:
It might be better style and less code to have two actions, with a not or no not.